nebula-logger/main/logger-engine/classes/FlowLogger.cls

```diff
@@ -42,14 +43,18 @@ public inherited sharing class FlowLogger {
             // Set the logging level if it's blank
             if (String.isBlank(this.loggingLevelName)) {
                 if (String.isNotBlank(this.faultMessage)) {
-                    this.loggingLevelName = 'ERROR';
+                    this.loggingLevelName = LoggingLevel.ERROR.name();
                 } else {
-                    this.loggingLevelName = 'DEBUG';
+                    this.loggingLevelName = LoggingLevel.DEBUG.name();
                 }
             }
 
-            this.loggingLevel = Logger.getLoggingLevel(this.loggingLevelName);
-            this.logEntryEventBuilder = Logger.newEntry(this.loggingLevel, this.message).setTopics(this.topics);
+            // In Flow, using comma-separated String is easier than List<String>
+            // but LogEntryEventBuilder.addTags() uses List<String>, so FlowLogger handles the conversion work
+            List<String> parsedTags = this.tagsString?.replace(', ', ',').replace(' ,', ',').split(',');
```

Comment:
not sure which would be more effective, but you could use `( ,)|(,)|(, )` to do this in one replace